python/py-emcee/Portfile

```diff
@@ -17,23 +17,17 @@ long_description    emcee is a stable, well tested Python implementation of \
                     Monte Carlo (MCMC) proposed by Goodman & Weare (2010). The \
                     code is open source and has already been used in several \
                     published projects in the astrophysics literature.
-homepage            https://github.com/dfm/emcee
 
-platforms           darwin
 supported_archs     noarch
 license             MIT
 
 checksums           rmd160  9d877a80ce5a19b8c442d225a27a57fe548a14f0 \
                     sha256  afb252b304051ca7a936e81adfa69c92b37fdafd8d3be95e920059f08dcf2d00 \
                     size    769439
 
-python.versions     27 35 36 37 38 39 310
+python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
-    if {${python.version} >= 36} {
-        conflicts   py${python.version}-emcee3
```

Comment:
Since we have dropped support for Python <= 3.6, the condition in the `if` statement is always true, so the `conflicts` statement needs to be kept.

Reply:
good catch, I missed that one - fixed locally!

Reply:
It's not showing up in GitHub yet...

Reply:
correct, as I said “fixed locally” and I haven’t pushed yet to this PR